code.user.js

```diff
@@ -1259,83 +1311,90 @@
                 },
                 function() {
                     logDOM('Could not retrieve the inventory...');
-                });
+                }
+            );
         }
 
         function sellAllDuplicateItems() {
-            loadAllInventories().then(function () {
-                var items = getInventoryItems();
-                var marketableItems = [];
-                var filteredItems = [];
+            loadAllInventories().then(
```

Comment:
Just commenting on this once (Chome hangs for a whole minute here), can we change all these callbacks to arrow functions?

Reply:
This? https://eslint.org/docs/latest/rules/prefer-arrow-callback

Reply:
Yes, thanks, this looks much better.